addons/sale_project/static/tests/tours/project_create_sol_tour.js

```diff
@@ -0,0 +1,98 @@
+import { markup } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+import { _t } from "@web/core/l10n/translation";
+
+registry.category("web_tour.tours").add('project_create_sol_tour', {
+    test: true,
+    url: "/web",
+    rainbowManMessage: _t("Congratulations! You have successfully created a sales order item on the fly from a Project."),
+    steps: () => [stepUtils.showAppsMenuItem(),
+{
+    trigger: ".o_app[data-menu-xmlid='project.menu_main_pm']",
+    position: "bottom",
+}, {
+    trigger: ".o-kanban-button-new",
+    extra_trigger: ".o_project_kanban",
+    content: markup(_t("Let\'s create your first <b>project</b>.")),
+    position: "bottom",
```

Comment:
No need since the purpose of that tour is just for testing to be sure the feature works. 🙂 

(same comment for all `position` attribute in the steps of that tour)